modules/apps/27-interchain-accounts/types/keys.go

```diff
@@ -8,12 +8,15 @@ const (
 	// ModuleName defines the interchain accounts module name
 	ModuleName = "interchainaccounts"
 
-	// VersionPrefix defines the current version for interchain accounts
-	VersionPrefix = "ics27-1"
-
-	// PortID is the default port id that the interchain accounts module binds to
+	// PortID is the default port id that the interchain accounts host submodule binds to
 	PortID = "interchain-account"
```

Comment:
Why don't we make the PortID of the host just `ics27`? So that it's a bit more similar to the PortID in the controller.

Reply:
I'm not opposed

Reply:
This would be a spec change, so maybe it can be discussed there and we can do a followup?

Reply:
No issues changing it